rwandaemr-installer/src/main/resources/scripts/install.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+## load variables from the vars.txt file
+source vars.txt
+
+create_target_dir(){
+    if [ "$env_type" == "demo" -o "$env_type" == "prod" -o "$env_type" == "test" ]; then
+      mkdir -p /opt/${parent_dir}/distribution
+    else
+      mkdir -p ${parent_dir}
+      mkdir -p ${parent_dir}/distribution
+    fi
```

Comment:
For example here.  You've already (in the download-distribution.sh script) switched on these environments to set up the TARGET_DIR.  you should just be able to re-use that specification here.  you just need that TARGET_DIR definition to happen at a top level (eg. in vars.txt), and then you can use it here without the switch statement